app/src/main/res/values-ro/strings.xml

```diff
@@ -827,12 +682,14 @@
     <string name="go_to_full_product_page">Go to product</string>
     <string name="no_additive_data_available_for_this_product">No additive data available for this product</string>
     <string name="no_nutrient_information_available_for_this_product">No nutrient information available for this product</string>
-    <string name="error_duplicate_listname">You already have a list with this name</string>
+    <string name="error_duplicate_<<<<<<< unused-resources
```

Comment:
@RajaVamsi11 is it a merge error ?

Reply:
yes

Reply:
Did not notice that. Thanks for committing.